### ISSUE TITLE
Close Focus when Original Element Closed

### ISSUE THREAD

**Issue by bmrgould** [gctools-outilsgc/gccollab-mobile#344]
SR specific.
When closing a new component opened from the More Options popover, the focus cannot be set properly (the original element was removed when the More Options popover closed). Needs to be provided context about what opened the previous element. 

Change to ID/Class based reference when original object not focus-able.

**Comment by bmrgould** [comment]
Situations where a component opens a component:

1. MoreOptions -> UserSheet
2. MoreOptions -> Report (Dialog)
3. UserSheet -> Send Message (Dialog)

One is solved by the PR.
Two and Three should be solved by having dedicated pages for Reporting and New Messages.

**Comment by bmrgould** [comment]
Of the three situations, first is solved, with a system for any future needs to handle it. 
Two and Three are not solved, but moved to being parts of other issues.
